fastlane_core/lib/fastlane_core/tool_collector.rb

```diff
@@ -2,16 +2,49 @@ module FastlaneCore
   class ToolCollector
     HOST_URL = "https://fastlane-enhancer.herokuapp.com"
 
+    # This is the original error reporting mechanism, which has always represented
+    # either controlled (UI.user_error!), or uncontrolled (UI.crash!, anything else)
+    # exceptions.
+    #
+    # Thus, if you call `did_crash`, it will record the failure both here, and in the
+    # newer, more specific `crash` field.
```

Comment:
It would be great if we could add the type of this variable here, so it easier to understand in what format the information is stored.